mewarpx/mewarpx/utils_store/profileparser.py

```diff
@@ -0,0 +1,158 @@
+import os
+import sys
+import re
+import json
+
+
+class FullProfile(object):
+    """
+    Class for parsing, storing and writing the full profile data
+    """
+    def __init__(self, lines, stdout_path, write_dir=None):
+        self.lines = lines
+        self.stdout_path = stdout_path
+        self.write_dir = write_dir
+
+        if self.write_dir is None:
+            if "/" in self.stdout_path:
+                self.write_dir = self.stdout_path[0 : self.stdout_path.rindex("/")]
+            else:
+                self.write_dir = "./"
```

Comment:
I believe this can be achieved more cleanly with
```suggestion
            self.write_dir = os.path.dirname(self.stdout_path)
```
but you should confirm that (and I guess you need the extra import).